src/bp-core/bp-core-functions.php

```diff
@@ -5286,3 +5286,17 @@ function bp_is_admin( $screen_id = '' ) {
 
 	return $screen_id === $bp->admin->current_screen;
 }
+
+/**
+ * Common function to check if bp_classic is active or not.
+ *
+ * @since 15.0.0
+ *
+ * @return bool True if bp_classic is active, false otherwise.
+ */
+function is_bp_classic_active() {
+    if ( function_exists( 'bp_classic' ) || is_plugin_active( 'bp-classic/bp-classic.php' ) ) {
+        return true;
+    }
+    return false;
```

Comment:
```suggestion
    return function_exists( 'bp_classic' ) || is_plugin_active( 'bp-classic/bp-classic.php' );
```
I believe we can save 2 lines of code 😅

Reply:
@imath Thanks for reviewing this. I have implemented the suggestions please check.